nengo_spinnaker/operators/lif.py

```diff
@@ -259,8 +264,8 @@ def write_subregion_to_file(self, fp, vertex_slice):
             self.n_output_dimensions,
             n_neurons,
             self.machine_timestep,
-            int(self.t_ref // self.dt),
-            tp.value_to_fix(self.dt / self.t_rc),
+            int(self.t_ref // self.dt),  # tau_ref expressed as in ticks
+            tp.value_to_fix(-np.expm1(-self.dt / self.t_rc) * (1.0 - 2**-17)),
```

Comment:
I'm relatively sure that math is right, but it'd be nice to have a plot of what happens to the tuning curve as you change that fudge factor.